larray/util/misc.py

```diff
@@ -631,8 +631,8 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 # deprecate_kwarg is derived from pandas.util._decorators (0.21)
-def deprecate_kwarg(old_arg_name, new_arg_name, mapping=None, stacklevel=2):
-    if not isinstance(mapping, dict):
+def deprecate_kwarg(old_arg_name, new_arg_name, mapping=None, fun=None, stacklevel=2):
```

Comment:
I prefer using func instead of fun. I admit it is less fun 😀 ... but it's less ambiguous and more standard.

Reply:
I started with func but if you look some lines below, you will see : 
```python
     def _deprecate_kwarg(func):
         @wraps(func)
```

Reply:
Didn't see that, but then a more explicit name would both help readability outside the decorator and inside it (help differentiate the two func). Names that come to mind: converter, arg_converter, arg_func, converter_func, ...? I think my preferred is arg_converter.